prjxray/roi.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+
+class Roi(object):
+    def __init__(self, tilegrid_file, x1, x2, y1, y2):
+        self.tilegrid_file = tilegrid_file
+        self.tilegrid = None
+        self.x1 = x1
+        self.x2 = x2
+        self.y1 = y1
+        self.y2 = y2
+
+    def tile_in_roi(self, tilej):
```

Comment:
instead of using the JSON directly, is there a Tile object that should be used instead?